src/edge_op.cpp

```diff
@@ -235,6 +235,9 @@ void Manifold::Impl::CleanupTopology() {
  * meshes, thus decreasing the Genus(). It only increases when meshes that have
  * collapsed to just a pair of triangles are removed entirely.
  *
+ * Verts with index less than firstNewVert will be left uncollapsed. This is
+ * zero by default so that everything can be collapsed.
```

Comment:
When do we use this zero default? We should probably explain this mesh simplification behavior in the public documentation as well.

Reply:
Everywhere except the Boolean - mesh ingestion, AsOriginal, etc. Hmm, come to think of it, the issue of decimating input planar geometry that doesn't intersect anything isn't really fixed, since we're still doing it on mesh import and other places. Perhaps those should all be restricted to short edge collapse? I think I'll do this in a separate PR.

Reply:
Sounds good.